torch/quantization/fx/_equalize.py

```diff
@@ -40,18 +43,16 @@ def __init__(self, dtype=torch.quint8, qscheme=torch.per_tensor_affine,
         if qscheme not in {torch.per_tensor_affine, torch.per_tensor_symmetric}:
             raise TypeError("Input qscheme must be per-tensor")
 
+        self.dtype = dtype
+
         self.input_obs = PerChannelMinMaxObserver(ch_axis=1, dtype=dtype,
                                                   qscheme=qscheme,
                                                   quant_min=quant_min,
                                                   quant_max=quant_max,
                                                   factory_kwargs=factory_kwargs)
 
         if output_obs is None:
-            self.output_obs = MinMaxObserver(dtype=dtype,
-                                             qscheme=qscheme,
-                                             quant_min=quant_min,
-                                             quant_max=quant_max,
-                                             factory_kwargs=factory_kwargs)
+            self.output_obs = MinMaxObserver
```

Comment:
nit: there could be some confusion if `input_obs` refers to an observer instance but `output_obs` refers to an observer class type.  Some strategies to improve this could be to name the variables differently (i.e. `output_obs_cls`), specify the types, store the instance instead of the class, etc.

Reply:
That makes sense; I will fix that!